packages/dapp/src/components/utility/Tabs.tsx

```diff
@@ -1,17 +1,32 @@
 import * as React from "react";
 import styled from "styled-components";
 
-const StyledUL = styled.ul`
-  border-bottom: 1px solid #aaa;
-  margin: 0 0 10px;
-  padding: 0;
+const TabContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
 `;
 
+const TabNav = styled.div`
+  margin: 0 auto 50px;
+  &.listings-nav {
```

Comment:
If we need DApp-specific styles, I think we can import the `Tabs` component from `@joincivil/components` into the DApp and extend it.

Something like this should work.

```
const ListingsNavTabs = Tabs.extend`
  <specific styles here>;
`;

<ListingsNavTabs>
  <Tab />
</ListingNavTabs>
```